3725/1/Gemfile

```diff
@@ -0,0 +1,11 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+group :development do
+  gem 'httparty'
+  gem 'mechanize'
+  gem 'nokogiri'
+  gem 'octokit'
+  gem 'reek'
+  gem 'rubocop'
+  gem 'terminal-table'
```

Comment:
Read this: https://bundler.io/v1.3/rationale.html
It is better to lock versions of gems, so as not to catch problems with their updates.